src/custom-element.js

```diff
@@ -55,9 +55,17 @@ const RETURN_TRUE = () => true;
  */
 let templateTagSupported;
 
-/** @type {!Array} */
+/** @type {!Array<AmpElement>} */
 export const stubbedElements = [];
 
+/**
+ * Extensions which have failed to load, making their elements unresolvable.
+ * If null, then any remaining elements which don't immediately have their
+ * implClass available are marked unresolvable.
+ * @type {Array<string>|null}
+ */
+let unresolvableExtensions = [];
```

Comment:
Can we somehow simplify this and/or make it more self-documenting?
As-is, this array represents all the elems which failed to load or that the page has finished loading. That combination of uses is a bit confusing.

I've got two potential ideas:
1. When the page finishes loading, populate the array with all of the remaining (failed) scripts. Potentially make this a `Set` as well.
2. Have two separate variables and a helper function that coordinates the logic.

Reply:
Converted to a `Set`, and now insert/check against `'*'` when all elements fail.

Reply:
Love it